emrichen/context.py

```diff
@@ -25,7 +26,7 @@ def enrich(self, value):
             return value
 
         elif isinstance(value, Mapping):
-            out = {}
+            out = (OrderedDict() if isinstance(value, OrderedDict) else {})
```

Comment:
would unconditionally `OrderedDict` break something?

Reply:
Probably not.